test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -155,7 +155,7 @@ def run_debug_test(self, model, train, batch_size, state_dict=None,
 
     def run_actual_test(self, model, train, batch_size, state_dict=None,
                         input=None, use_gpu=True, rtol=0.001, atol=1e-7,
-                        example_outputs=None):
+                        example_outputs=None, do_constant_folding=False):
```

Comment:
I am wondering if we enable the flag, how many tests will pass?

@spandantiwari do you want to give it a try?

Reply:
@houseroad, maybe as the next PR. :-)

Reply:
i mean locally first, i am wondering how stable it is :-)